templates/templated_email/html/source/dashboard/staff/promote_customer.mjml

```diff
@@ -0,0 +1,30 @@
+<mjml>
+  <mj-head>
+    <mj-include path="../../shared/styles" />
+  </mj-head>
+  <mj-body>
+    <mj-container>
+      <mj-section css-class="no-display">
+        <mj-column>
+          <mj-text>
+            {% load i18n %}
+          </mj-text>
+        </mj-column>
+      </mj-section>
+      <mj-include path="../../shared/header" />
+      <mj-section>
+        <mj-column>
+          <mj-text  font-size="16px">
+            {% trans "Hi!" context "Promote customer to staff member confirmation e-mail text" %}
+          </mj-text>
+          <mj-text>
+            {% blocktrans context "Promote customer to staff member confirmation e-mail text" %}
+              You're receiving this e-mail because you have been promoted to staff member at {{ domain }}.
```

Comment:
This template (as well as it's plain text version) could also contain a link to the dashboard, so that receiver of this email could quickly log in there.

Reply:
Ok, I'll add it.